tasks/marketplacesvm-push-disk-images/marketplacesvm-push-disk-images.yaml

```diff
@@ -0,0 +1,143 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: marketplacesvm-push-disk-images
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push disk images to Cloud Marketplaces
+  params:
+    - name: snapshotPath
+      type: string
+      description: |
+        Path to the JSON string of the mapped snapshot spec in the data workspace.
+        It must be processed by the "apply-mapping" task first.
+    - name: cloudMarketplacesSecret
+      type: string
+      description: Env specific secret containing the marketplaces credentials.
+    - name: concurrentLimit
+      type: string
+      description: The maximum number of images to be pulled at once.
+      default: 3
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot spec json file resides
+  steps:
+    - name: pull-and-push-images-to-marketplaces
+      image: quay.io/konflux-ci/release-service-utils:7f7a156a835c773bbcd7e5d7e44df2f573db14f2
+      env:
+        - name: CLOUD_CREDENTIALS
+          valueFrom:
+            secretKeyRef:
+              name: $(params.cloudMarketplacesSecret)
+              key: key
+      script: |
+        #!/usr/bin/env bash
+        set -ex
+
+        # Setup required variables
+        SNAPSHOT_JSON=$(jq -c '.' "$(workspaces.data.path)/$(params.snapshotPath)")
+        STARMAP_MAPPING=$(jq -c '.components[0].starmap' <<< "$SNAPSHOT_JSON")
+        STARMAP_MAPPING_FILE=$(workspaces.data.path)/starmap.yaml
```

Comment:
This isn't a good idea. It will cause issues with concurrent runs if the same workspace is used. I'd recommend something like this instead https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/create-pyxis-image/create-pyxis-image.yaml#L94

Reply:
Ok!